scripts/prepare_release.sh

```diff
@@ -50,11 +51,12 @@ print_usage_and_exit() {
 
 	cat <<- EOF
 	Usage:
-	  $ $(basename "$0") <version> [-h] [-v]
+	  $ $(basename "$0") <version> [banch-with-fix] [-h] [-v]
 	  Current version: $(cut -d' ' -f3 < "${base_dir}/Configuration/Version.xcconfig")
 
 	Options:
-	  -h         Make hotfix release
+	  -h         Make hotfix release. Requires the version to be the one to hotfix, and a branch with the fix as the second parameter
```

Comment:
do we provide any error info if someone forgets to provide a branch name?

Reply:
No, it fails silently (although at least in a fairly obvious way:
"Merging fix branch ... %                                                     
ellesullivanwork@Elles-MBP iOS %"

Is there an easy way to do it?

Reply:
I've added a check based on the number of arguments:

"Usage:
 ...
💥 Error: Wrong number of arguments. Did you specify a fix branch?"